app.ts

```diff
@@ -2,9 +2,16 @@ import { NextFunction, Request, Response } from 'express';
 
 import express from 'express';
 import routes from './routes/api';
+import cors from 'cors';

```

Comment:
just wondering... are the changes from main branch supposed to show up here?

Reply:
huh, I'm not entirely sure. It looks like its coming from a commit you made which is weird that it'd be attached to this PR. 

I'm not sure what the next steps would be if it'll mess up the main to leave this here. Worse case I can redo the PR?

Reply:
I was able to fix the pr, the main changes are no longer in the pr